snapwell/wellpath.py

```diff
@@ -59,11 +61,11 @@ def setRkb(self, rkb):
         try:
             t = float(rkb[0]), float(rkb[1]), float(rkb[2])
             self._rkb = t
-        except ValueError:
+        except ValueError as e:
```

Comment:
Can we replace this with `err` for readability?

Reply:
Can do